src/nxt_port.h

```diff
@@ -115,6 +117,7 @@ typedef enum {
     _NXT_PORT_MSG_APP_RESTART     = nxt_port_handler_idx(app_restart),
     _NXT_PORT_MSG_STATUS          = nxt_port_handler_idx(status),
 
+    _NXT_PORT_MSG_CLIENT_DISCONNECTED = nxt_port_handler_idx(client_disconnected),
```

Comment:
Hi @gourav-kandoria thanks for the PR!

So just to clarify. This is about when the client _doesn't_ do a clean shutdown of the socket (via either close(2) or shutdown(2)), but rather the connection is lost for some other reason, client crash, network issue etc...?

If so, I wonder if these would be better named `_CLIENT_ERROR`?

Actually that ties up with the fact we check the `error` member of `nxt_http_request_t`